src/NewsService.Business/GetNewsCommand.cs

```diff
@@ -15,29 +28,128 @@ public class GetNewsCommand : IGetNewsCommand
     private readonly INewsRepository _repository;
     private readonly INewsResponseMapper _mapper;
     private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IRequestClient<IGetUsersDataRequest> _rcGetUsers;
+    private readonly IRequestClient<IGetDepartmentsRequest> _rcGetDepartments;
+    private readonly IDepartmentInfoMapper _departmentInfoMapper;
+    private readonly ILogger<GetNewsCommand> _logger;
+
+    private UserData GetAuthor(Guid userId, List<string> errors)
+    {
+      string errorMessage = "Cannot get author now. Please try again later.";
+      const string logMessage = "Cannot get author with ids: {userId}.";
+
+      try
+      {
+        IOperationResult<IGetUsersDataResponse> response = _rcGetUsers.GetResponse<IOperationResult<IGetUsersDataResponse>>(
+          IGetUsersDataRequest.CreateObj(new List<Guid> { userId })).Result.Message;
+
+        if (response.IsSuccess)
+        {
+          return response.Body.UsersData.FirstOrDefault();
+        }
+
+        _logger.LogWarning(logMessage, string.Join(", ", userId));
```

Comment:
string.Join with guid? what should happened?

Reply:
done